src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,15 +14,388 @@ tree_item_index: 1100
+                "s3:ListBucketMultipartUploads"
+            ],
+            "Resource": "arn:aws:s3:::yourbucketname"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:DeleteObject",
+                "s3:AbortMultipartUpload",
+                "s3:ListMultipartUploadParts",
+                "s3:ListBucketMultipartUploads"
+            ],
+            "Resource": "arn:aws:s3:::yourbucketname/*"
+        }
+    ]
+}
+```
+
+Make sure that all configuration values are set in the `nuxeo.conf`:
+
+```
+nuxeo.s3storage.transient.bucket=
+nuxeo.aws.accessKeyId=
+nuxeo.aws.secretKey=
+nuxeo.aws.region=
+nuxeo.s3storage.bucket=
+```
+{{#> callout type='note'}}
+
+Since the work in [NXP-29548](https://jira.nuxeo.com/browse/NXP-29548) 'nuxeo.s3storage.useDirectUpload=true' and 'nuxeo.s3storage.transient.roleArn=' (documented as part of the previous version configuration) are no longer necessary **for the Nuxeo Aspera Connector** and can be removed, **if you only use these values for the Nuxeo Aspera Connector, and not for any other reason**.  See upgrade notes.
+{{/callout}}
+
+#### Aspera Nuxeo Configuration
+
+Add the aspera access keys in `nuxeo.conf`:
+
+**Example**:
+
+```
+## is the same for the 2 nodes
+aspera.node.url=https://ats-aws-us-east-1.aspera.io
+aspera.node.port=443
+
+### ACCESS KEY ON UPLOAD NODE LINKED TO TS S3 BUCKET
+aspera.access.key.id=
+### SECRET KEY ON UPLOAD NODE LINKED TO TS S3 BUCKET
+aspera.access.key.secret=
+
+### ACCESS KEY ON DOWNLOAD NODE LINKED TO TMAIN S3 BUCKET
+aspera.download.access.key.id=
+### SECRET KEY ON DOWNLOAD NODE LINKED TO MAIN S3 BUCKET
+aspera.download.access.key.secret=
+```
+
+### Functional Overview
+The **Nuxeo Aspera Connector** uses **Transfers** to add content to the Nuxeo system.  Transfers are “wrappers” or “buckets” for the files/folders you add to the Nuxeo system.
+
+#### Transfer steps
+The process of adding content is divided into 2 steps: **Upload** and **Create**. 
+In the first step, the user **uploads** content to the transfer bucket. While in this step, it may be possible, depending on the chosen “transition type”,  for a user to add files to the bucket and add properties that will apply to the content. 
+Once content and any metadata have been added, the transfer can be completed and the content is then **created** in the system.
+
+#### Transfer transition types
+Transitioning between the “upload” and “create” steps can be achieved in 2 ways: **automatically (auto-create)** or **manually**. Users can choose which option to use when creating the transfer from the transfer dashboard.   It is possible to change from manual to auto-create after you start adding content to the transfer.  **Once you set the to auto-create, however, the transfer will behave as if you started in that way, and some limitations around modifying the transfer will apply**.
```

Comment:
It is the other way around, right? 🤔 

```suggestion
Transitioning between the “upload” and “create” steps can be achieved in 2 ways: **automatically (auto-create)** or **manually**. Users can choose which option to use when creating the transfer from the transfer dashboard.   It is **not** possible to change from manual to auto-create after you start adding content to the transfer.  **Once you set the to auto-create, however, the transfer will behave as if you started in that way, and some limitations around modifying the transfer will apply**.
```

Reply:
oy, thanks @nmpcunha  that's why we have reviews...

Reply:
@nmpcunha reading through this...it is as I originally wrote it...you can start with the transfer set to manually complete and keep it there after adding content OR, if you choose, once you started adding content, you can switch to autocreate, BUT from that point, any content you add will be your "last"; the transfer will complete upon THAT content being uploaded.

Reply:
Not sure If I understood your point, TBH 😕 
This sentence doesn't seem clear to me: `It is possible to change from manual to auto-create after you start adding content to the transfer`

Reply:
@nmpcunha I'm almost positive that we could at one time.  I just tested though, and you're correct 😄 .  at one time, I thought we had that so that if you had folders at different "root" locations in your finder/explorer, you could do part (grab one root), then change to auto and then get the other root location.  I will modify the documentation 👍